plugins/test/unit/plugins/importers/yum/test_purge.py

```diff
@@ -288,3 +289,30 @@ def test_retain_old(self, mock_remove_old_versions):
         purge.purge_unwanted_units(self.metadata_files, self.conduit, self.config)
 
         mock_remove_old_versions.assert_called_once_with(3, self.conduit)
+
+
+class RemoveUnitDuplicateNevra(TestPurgeBase):
+
+    @mock.patch.object(purge, 'RepoUnitAssociationManager', autospec=True)
+    @mock.patch.object(purge, 'UnitAssociationCriteria', autospec=True)
+    def test_remove_unit_duplicate_nerva(self, mock_criteria, mock_association):
+        unit_key = {'name': 'test-nevra', 'epoch': 0, 'version': 1, 'release': '23',
+                    'arch': 'noarch', 'checksum': '1234abc', 'checksumtype': 'sha256'}
+        type_id = 'rpm'
+        repo_id = 'test-repo'
+        expected_filters = set([('arch', 'noarch'), ('epoch', 0), ('name', 'test-nevra'),
+                            ('release', '23'), ('version', 1)])
```

Comment:
You should align this line with the one above for PEP-8 so that it doesn't cause a problem when merging forward.